charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Datadog changelog
 
+## 2.15.2
+* Fix Cilium network policys.
```

Comment:
```suggestion
* Fix Cilium network policies.
```

Reply:
Thanks have changed it and upped the versions again